privacyidea/cli/pitokenjanitor/utils/importtokens.py

```diff
@@ -95,3 +97,43 @@ def pskc(pskc_file, preshared_key, validate_mac):
               f" because they could not be validated: {not_parsed_tokens}")
     print(f"Successfully imported {success} tokens.")
     print(f"Failed to import {failed} tokens: {failed_tokens}")
+
+@importtokens_cli.command('privacyidea')
+@click.argument('file', nargs=1, type=click.File())
+@click.argument('key', nargs=1, type=str)
+def import_token_from_privacyidea(file, key):
+    """
+    Import tokens from a PrivacyIDEA token file.
+
+    FILE is the file containing the tokens to be imported.
+    KEY is the encryption key used to decrypt the tokens.
+    """
+    try:
+        decrypt_key = Fernet(key)
+    except Exception as e:
+        click.echo(f"Invalid encryption key format. Please ensure the key is valid. Details: {e}")
+        return
+
+    try:
+        token_data = file.read()
+    except Exception as e:
+        click.echo(f"Error reading file: {e}")
+        return
+
+    try:
+        decrypted_data = decrypt_key.decrypt(token_data)
+    except Exception as e:
+        click.echo(f"Error decrypting token data: {e}")
+        return
+
+    try:
+        token_list = decrypted_data.decode("utf-8")
+    except UnicodeDecodeError as e:
+        click.echo(f"Error decoding token data: {e}")
+        return
+
+    ret = import_tokens(token_list)
```

Comment:
We should catch possible exceptions here as well to inform the user with a stacktrace